can/interfaces/__init__.py

```diff
@@ -29,4 +28,4 @@
     for interface in iter_entry_points('can.interface')
 })
 
-VALID_INTERFACES = frozenset(list(BACKENDS.keys()) + ['socketcan'])
+VALID_INTERFACES = frozenset(list(BACKENDS.keys()) + ['socketcan_native', 'socketcan_ctypes'])
```

Comment:
Well, we could then just remove the special case all together. Although this would break compatibility with older versions, it would clean up a lot of code.